pages.it/common/git-mv.md

```diff
@@ -0,0 +1,16 @@
+# git mv
+
+> Sposta o rinomina file e aggiorna l'indice Git.
+> Maggiori informazioni: <https://git-scm.com/docs/git-mv>.
+
+- Sposta i file nella repository e aggiungi l'operazione al commit successivo:
+
+`git mv {{percorso/al/file}} {{nuovo/percorso/al/file}}`
+
+- Rinomina i file e aggiungi l'operazione al commit successivo:
+
+`git mv {{percorso/al/file}} {{percorso/al/file_rinominato}}`
```

Comment:
This is not the 2nd example in the English page:
```suggestion
`git mv {{file}} {{file_rinominato}}`
```

Reply:
In my understanding, it actually is. The second example in the English page refers to a filename, which it's a path/to/file. You might think of a mapping `filename` -> `path/to/file` (i.e., `percorso/al/file`), and `new_filename` -> `path/to/renamed_file` (i.e., `percorso/al/file_rinominato`).

But if you think that is more clear stressing the name change only, I'll update my translation :)

Reply:
In Linux terms, the 1st and 2nd examples are identical, but I think the idea is only say `as user you can rename a file within the same folder`. You can await the review from another collaborator cc @sbrl @owenvoke

Reply:
Hrm, I don't know italian so I can't really comment here.

Reply:
me neither, but I think @glenacota is not talking about something specific to it language, right?

Reply:
> me neither, but I think @glenacota is not talking about something specific to it language, right?

exactly, I have the same doubt for the English version. But for the sake of this PR, I can translate the token literally ;)